src/CssInliner.php

```diff
@@ -172,9 +172,13 @@ public function inlineCss(string $css = ''): self
         $cssWithoutComments = $this->removeCssComments($combinedCss);
         list($cssWithoutCommentsCharsetOrImport, $cssImportRules)
             = $this->extractImportAndCharsetRules($cssWithoutComments);
+        list($cssWithoutCommentsCharsetOrImportAndFontFaces, $cssFontFaces)
+            = $this->extractFontFaces($cssWithoutCommentsCharsetOrImport);
+
+        $uninlinableCss = $cssImportRules . $cssFontFaces;
 
         $excludedNodes = $this->getNodesToExclude();
-        $cssRules = $this->parseCssRules($cssWithoutCommentsCharsetOrImport);
+        $cssRules = $this->parseCssRules($cssWithoutCommentsCharsetOrImportAndFontFaces);
```

Comment:
I don't think the variable name should contain both "or" and "and" - perhaps `$cssWithoutCommentsCharsetImportOrFontFace`?

Reply:
I agree. I renamed it as well as the name of the extraction method, just to be consistent.